src/pages/discover.js

```diff
@@ -8,14 +8,51 @@ import withIntl from '../lib/withIntl';
 
 import { getBaseApiUrl } from '../lib/utils';
 import { queryString } from '../lib/api';
-
+import styled from 'styled-components';
+import Link from 'next/link';
+import { FormattedMessage } from 'react-intl';
 import { Box, Flex } from '@rebass/grid';
-import CollectiveCard from '../components/CollectiveCard';
+
 import Container from '../components/Container';
 import Page from '../components/Page';
 import { H1, P } from '../components/Text';
 import LoadingGrid from '../components/LoadingGrid';
 import Pagination from '../components/Pagination';
+import SearchForm from '../components/SearchForm';
+
+import CollectiveCard from '../components/CollectiveCard';
+import PledgedCollectiveCard from '../components/PledgeCollectiveCard';
+
+const NavList = styled(Flex)`
+  list-style: none;
+  min-width: 20rem;
+  text-align: right;
+`;
+
+const SelectWrapper = styled.select`
+  background: white;
+  padding: 5px;
+  border: 1px solid silver;
+`;
+
+const Nava = styled.a`
+  color: grey;
+  :hover {
+    color: blue;
+    cursor: pointer;
+  }
+`;
+const SearchFormContainer = styled(Box)`
+  max-width: 30rem;
+  min-width: 83rem;
```

Comment:
Why is `min-width` bigger than `max-width`? I don't think this will work on mobile